firmware/ui/src/SerialStm.cc

```diff
@@ -38,15 +38,8 @@ bool SerialStm::ReadyToWrite()
 
 void SerialStm::Write(unsigned char* buff, const unsigned short buff_sz)
 {
-    // tx_free = false;
     uint8_t start_byte[1] = { 0xFF };
-
-    const auto& str_size = "[STM] " + std::to_string(buff_sz) + "\n";
-    HAL_UART_Transmit(&huart1, reinterpret_cast<const uint8_t*>(str_size.c_str()), str_size.size(), HAL_MAX_DELAY);
-
     HAL_UART_Transmit(uart, start_byte, 1, HAL_MAX_DELAY);
-
-    // HAL_UART_Transmit_IT(uart, buff, buff_sz);
```

Comment:
This should still be in the codebase, as its originally supposed to be an interrupt call, but was removed for testing.

Reply:
Re-add in this comment and add a TODO re-implement interrupt.